lib/VideoPlayer/VideoPlayerAccessibilitySupport.js

```diff
@@ -0,0 +1,58 @@
+var
+	kind = require('enyo/kind');
+
+var
+	$L = require('../i18n');
+
+/**
+* @name VideoPlayerAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	initAccessibility: kind.inherit(function (sup) {
+		return function (props) {
+			sup.apply(this, arguments);
+			this.$.jumpBack.set('accessibilityLabel', $L('jumpBack'));
+			this.$.rewind.set('accessibilityLabel', $L('rewind'));
+			this.$.fastForward.set('accessibilityLabel', $L('fastForward'));
+			this.$.jumpForward.set('accessibilityLabel', $L('jumpForward'));
+			this.$.moreButton.set('accessibilityLabel', $L('more'));
+		};
+	}),
+
+	/**
+	* @private
+	*/
+	updatePlayPauseButtons: kind.inherit(function (sup) {
+		return function (props) {
+			sup.apply(this, arguments);
+			if (this._isPlaying) {
```

Comment:
This could be a bit more efficient by choosing the label in the if block or via ternary op and assigning it after.

```
var label = this._isPlaying ? $L('Pause') : $L('Play');
this.$.fsPlayPause.set('accessibilityLabel', label);
this.$.ilPlayPause.set('accessibilityLabel', label);
```